### ISSUE TITLE
A11y_code.visualstudio.com_Docs User Guide_Emmet_ScreenReader: "Edit" link name should be more descriptive.

### ISSUE THREAD

**Issue by RohitJadhav-TCS** [microsoft/vscode-docs#2784]
“[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-manai%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
URL: [https://code.visualstudio.com](https://code.visualstudio.com)
Windows Version: Windows10
Browser Version:
Microsoft Edge 44.18362.1.0
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Screen Reader Versions : NVDA2019.1.1
MAS Violated: MAS 2.4.4

**Repro Steps:**
Navigate to the URL: [https://code.visualstudio.com/docs/editor/emmet](https://code.visualstudio.com/docs/editor/emmet)
follow below steps:
1.Launch the application URL: [https://code.visualstudio.com](https://code.visualstudio.com)
2.Navigate to "Docs" link.
3.From left navigation, select "User Guide".
4.Navigate to "Emmet" link and activate it.
5.Navigate to the main content.
6.Start the screen reader.
7.Navigate to the “Edit” link

**Actual:**
"Edit" link is present on almost every screen. Each link is having same name as "Edit" even those  links are navigating to different URLs.

This Issue is observed with Edge+Narrator and Chrome+NVDA
 
**Expected:**
If Link is present on all screens with same Name, then it should be more descriptive as it is navigating to different screens..    

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
Screen reader user will get confused while navigating through link.
This edit link is present on almost all screens so user will get confused that particular Edit Link is belongs to which screen. 

**MAS Reference:**
MAS 2.4.4 -
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={06a891af-210c-4c6d-bb3e-92babab209e7}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={06a891af-210c-4c6d-bb3e-92babab209e7})

**Attachment for Reference:**
[Code.visualstudio.com_Docs User Guide_Emmet_ScreenReader Edit link name should be more descriptive.pptx](https://github.com/microsoft/vscode-docs/files/3314370/Code.visualstudio.com_Docs.User.Guide_Emmet_ScreenReader.Edit.link.name.should.be.more.descriptive.pptx)

**Comment by RohitJadhav-TCS** [comment]
Similar Issue is observed in following screens:
1) Themes : https://code.visualstudio.com/docs/getstarted/themes
2) Continuous Integration :https://code.visualstudio.com/api/working-with-extensions/continuous-integration
3) Display Language :https://code.visualstudio.com/docs/getstarted/locales
4) Angular Tutorial :https://code.visualstudio.com/docs/nodejs/angular-tutorial
5) Java : https://code.visualstudio.com/docs/languages/java
6) MacOS :https://code.visualstudio.com/docs/setup/mac
